src/OpenTelemetry/Resources/ResourceBuilder.cs

```diff
@@ -30,15 +30,22 @@ private ResourceBuilder()
         {
         }
 
+        private static Resource DefaultResource { get; } = new Resource(new Dictionary<string, object>
+        {
+            [ResourceSemanticConventions.AttributeServiceName] = "unknown_service:"
```

Comment:
if processname is not available, we'll get "unknown_service:". what we need is "unknown_service" (without the : at the end.